MyBus/MyBus/MyBusMapView.swift

```diff
@@ -279,24 +279,25 @@ class MyBusMapView: MGLMapView{
             case first = 1.2
             case second = 1.8
             case third = 2.8
-            case fourth = 3.5
```

Comment:
I've left this code commented in case the product owner regrets his decision about not having a progressive zoom.